tests/test_templates/rules/S3LifecycleConfiguration/allowed_template_malformed_lifecycle_rules.yaml

```diff
@@ -0,0 +1,10 @@
+Resources:
+  OutputBucket:
+    Type: AWS::S3::Bucket
+    Properties:
+      BucketName: "foo"
+      AccessControl: BucketOwnerFullControl
+      LifecycleConfiguration:
+        # This is not valid for LifecycleConfiguration, but CFRipper will not parse it right now.
```

Comment:
This comment is misleading. Aren't we adding this to check if CFRipper raises a failure?

Reply:
We are just checking for the presence of this key (`LifecycleConfiguration`) as opposed to its value. A valid CF template would need a list of `Rules` under this key.